WordPress/Classes/Services/EditorSettingsService.swift

```diff
@@ -65,47 +65,50 @@ import Foundation
         }, failure: failure)
     }
 
-    func syncEditorSettingsForAllBlogs() {
-        let blogService = BlogService(managedObjectContext: managedObjectContext)
-        guard let blogs = blogService.blogsForAllAccounts() as? [Blog] else {
+    /// This method is intended to be called only after the db 87 to 88 migration
+    ///
+    func postAppWideEditorSettingToRemoteForAllBlogsAfterMigration(database: KeyValueDatabase = UserDefaults.standard) {
+        guard let api = apiForDefaultAccount else {
+            // SelfHosted non-jetpack sites won't sync with remote.
             return
         }
-        let blogsWithAccount = blogs.filter({ $0.account != nil })
-        let queue = SyncQueue()
-        for blog in blogsWithAccount {
-            queue.add { (next) in
-                self.syncEditorSettings(for: blog, success: {
-                    next()
-                }, failure: { (error) in
-                    DDLogError("Error saving editor settings: \(error)")
-                    next()
-                })
-            }
+
+        let isGutenbergEnabledAppWide = database.bool(forKey: GutenbergSettings.Key.appWideEnabled)
+        let editor: EditorSettings.Mobile = isGutenbergEnabledAppWide ? .gutenberg : .aztec
```

Comment:
I'm thinking on having this as a parameter, so this method is more generic and can be reused later if needed